modules/soplex/7.1.2/patches/add_build_dot_bazel.patch

```diff
@@ -0,0 +1,47 @@
+diff --git a/BUILD.bazel b/BUILD.bazel
+new file mode 100644
+index 00000000..f9d52aab
+--- /dev/null
++++ b/BUILD.bazel
+@@ -0,0 +1,41 @@
++# Description:
++# Soplex is a LP solver, a common tool in Operations Research.
++#
++# Copyright 2011 Google Inc.  All rights reserved.
++
++cc_library(
++    name = "libsoplex",
++    srcs = glob(
++        [
++            "src/soplex/*.cpp",
++            "src/soplex/external/fmt/*.cc",
++        ],
++        exclude = [
++            "src/git_hash.cpp",
++        ],
++    ),
++    copts = [
++        "-w",
++        "-fexceptions",
++        "-DGITHASH=\"NoGitInfo\"",
++    ],
++    features = ["-use_header_modules"],  # Incompatible with -fexceptions.
++    includes = [
++        "src",
++        "src/soplex/external",
```

Comment:
This seems dangerous as it leaks vendored `fmt` into all dependents. Maybe try using `implementation_deps` to contain the impact?